machine_management.adoc

```diff
@@ -0,0 +1,249 @@
+= Machine management
+
+toc::[]
+
+
+## Applying autoscaling to an {product-title} cluster
+
+[source,terminal]
+----
+$ oc create -f <filename>.yaml 
+----
+
+
+[source,terminal]
+----
+$ oc create -f <filename>.yaml 
+----
+
+## Creating infrastructure machine sets
+
+[source,terminal]
+----
+$ oc create -f <file_name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc adm taint nodes <node_name> <key>:<effect>
+----
+
+
+[source,terminal]
+----
+$ oc adm taint nodes node1 node-role.kubernetes.io/infra:NoSchedule
```

Comment:
Needs a declarative alternative

Reply:
This is an example of the previous command, in the same step. One declarative for both commands